eva/parser/evaql/evaql_parser.g4

```diff
@@ -71,10 +71,11 @@ createUdf
     : CREATE UDF
       ifNotExists?
       udfName
-      INPUT  createDefinitions
```

Comment:
Core of the change set. Added CATEGORY as a String Literal. We should also make change to query execution clause, where we pass CATEGORY instead of the UDF name